goodtables/checks/duplicate_row.py

```diff
@@ -24,7 +24,9 @@ def check_row(self, cells):
 
         # Get pointer
         try:
-            pointer = hash(json.dumps(list(cell.get('value') for cell in cells)))
+            values = list(cell.get('value') for cell in cells)
+            # https://github.com/frictionlessdata/goodtables-py/issues/329
+            pointer = hash(json.dumps(values + [len(values)]))
```

Comment:
@roll, I don't think this will avoid collisions.

```values + [len(values)]```

Since rows often have the same number of values.

Reply:
@davidpeckham 
Sure, thanks. I've fixed it already :smi